.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.0
+
+jobs:
+  execute-npm:
+    parameters:
+      npm-run-command:
+        type: string
+    docker:
+      - image: cimg/base:stable
+    environment:
+      SCRIVITO_TENANT: "1234567890abcdef1234567890abcdef"
+    steps:
+      - checkout
+      - node/install
+      - run:
+          name: Install correct npm version
+          command: |
+            echo "Node version"
+            node --version
+            echo ""
+            dpkg --compare-versions `npm -v` ge 7.10 || npm i -g npm@^7.10
+            echo ""
+            echo "NPM version"
+            npm --version
+      - node/install-packages
+      - run: npm run << parameters.npm-run-command >>
+
+workflows:
```

Comment:
* [ ] Did you consider `run-npm-command`? From the docs:
```yaml
workflows:
  run-npm-command:
    jobs:
      - node/run:
          npm-run: build
```

Reply:
I did. But I could not set `environment` for it. That is why I did implement the job `execute-npm`.